### PR TITLE
Support for oembed in embedded resources module

### PR DESCRIPTION
**Pull request by ludwiktrammer**

### CODE REVIEW

File: addons/website_embedded/models.py [hunk 1]
```diff
@@ -1,22 +1,70 @@
...
+                oembed_urls = oembed.discoverer.get_oembed_urls(self.resource_url)
+                if oembed_urls:
+                    oembed_response = oembed_consumer.get_first_oembed_response(
+                        oembed_urls,
+                        max_width=800,
+                        max_height=600
+                    )
+                    embed_code = oembed.renderer.render(self.resource_url, oembed_response)
+                    thumbnail_url = getattr(oembed_response, 'thumbnail_url', '')
+                if not (oembed_urls and embed_code and thumbnail_url):
```
Comment (KrzysiekJ) [inline addons/website_embedded/models.py]: This is tricky. If `oembed_urls` evaluates to `False`, then `embed_code` and `thumbnail_url` are not defined. Therefore putting `oembed_urls` is for the sake of preventing unbound variable errors. Maybe it is better to explicitly initialize `embed_code` and `thumbnail_url` with `None` (“explicit is better than implicit”)? Also why not reverse the order in `if`, so we can get rid of `not`?

Comment (ludwiktrammer) [inline addons/website_embedded/models.py]: Why do you mean by "reverse the order"?

Comment (KrzysiekJ) [inline addons/website_embedded/models.py]: You can write
```python
if oembed_urls and embed_code and thumbnail_url:
    self.embed_code = embed_code
    self.thumbnail_url = thumbnail_url
else:
    error_message = u'''Nie udało się pobrać kodu osadzenia (embed code) tego zasobu!'''
```

Comment (ludwiktrammer) [inline addons/website_embedded/models.py]: Done.

**Pull request merged**